tests/test_other.py

```diff
@@ -1670,6 +1670,7 @@ def test_libjpeg(self):
     self.emcc(test_file('jpeg_test.c'), ['--embed-file', 'screenshot.jpg', '-s', 'USE_LIBJPEG'], output_filename='a.out.js')
     self.assertContained('Image is 600 by 450 with 3 components', self.run_js('a.out.js', args=['screenshot.jpg']))
 
+  @disabled('https://github.com/emscripten-core/emscripten/issues/14255')
```

Comment:
Revert his part of the revert I think.. we don't want the tests disabled anymore

Reply:
Done